types/types.go

```diff
@@ -189,6 +191,24 @@ type Metrics struct {
 	SizeMemoryCache uint64
 }
 
+type PerModuleMetrics struct {
+	Hits uint32 `msgpack:"hits"`
+	Size uint64 `msgpack:"size"`
+}
+
+type PerModuleEntry struct {
+	Name    string
```

Comment:
Shouldn't this field be called Checksum?

Reply:
Renamed it. And also changed the type to `Checksum` since it's now not hex anymore but the raw byte array.